### PR TITLE
RA-559:Words on green 'Confirm' button to confirm new patient contact inf…

### PR DESCRIPTION
**Pull request by vishnuraom**
…o disappears

### CODE REVIEW

**Comment by dkayiwa** [comment]
@vishnuraom which of this and https://github.com/openmrs/openmrs-module-registrationapp/pull/12 should we take for RA-559?

**Comment by vishnuraom** [comment]
@dkayiwa Both of these PR's together will provide a fix for RA-559.

**Comment by dkayiwa** [comment]
@vishnuraom why don't you combine into one?

**Comment by vishnuraom** [comment]
@dkayiwa Both PR's belong to different modules.  Could you please point me to some resource where I could find the instructions to combine into one? I didn't do it before.

**Comment by dkayiwa** [comment]
Oh i see! Then you are right to have different pull requests.

File: omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js [hunk 1]
```diff
@@ -602,11 +602,13 @@ ConfirmationSectionModel.prototype.select = function() {
     })
 
     if (!hasData) {
-        this.questions[0].confirm.disable();
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js]: @vishnuraom the code you are replacing looks more generic than yours. Have you tried to investigate what was wrong with it? Could the problem be somewhere else instead?

Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js]: @vishnuraom did you see the above comment?

**Comment by vishnuraom** [comment]
@dkayiwa I didn't get any time to look into this recently because of my exams but when I first tried to solve the ticket,I tried to implement the changes proposed at http://stackoverflow.com/questions/1414365/disable-enable-an-input-with-jquery?rq=1  with https://github.com/openmrs/openmrs-module-uicommons/blob/master/omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js#L31
as the disable,enable functions are not working properly and it didn't work out.
So I replaced the code at https://github.com/openmrs/openmrs-module-uicommons/blob/master/omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js#L605

However I will look into it again once I get some time.

**Comment by dkayiwa** [comment]
@vishnuraom tell me to reopen when you get time. :)

**Pull request closed without merging**